cmd/configure/createprofile/createprofile.go

```diff
@@ -0,0 +1,64 @@
+package createprofile
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var CreateProfileCmd *cobra.Command
+
+func init() {
+	CreateProfileCmd = &cobra.Command{
+		Use:   "create-profile",
+		Short: "Create profile",
+		Long:  "Creates a new profile. Use this to create a profile with a name, host, and token.",
+		RunE:  createProfileHandler,
+	}
+
+	_ = CreateProfileCmd.Flags().String("name", "", "Name of profile.")
+	_ = CreateProfileCmd.Flags().String("host", "", "Host for profile.")
+	_ = CreateProfileCmd.Flags().String("token", "", "Token for profile.")
+
+	// Required flags
+	_ = CreateProfileCmd.MarkFlagRequired("name")
+	_ = CreateProfileCmd.MarkFlagRequired("host")
+	_ = CreateProfileCmd.MarkFlagRequired("token")
+}
+
+func createProfileHandler(cmd *cobra.Command, args []string) error {
+	name, err := cmd.Flags().GetString("name")
+	if err != nil {
+		return err
+	}
+
+	host, err := cmd.Flags().GetString("host")
+	if err != nil {
+		return err
+	}
+
+	token, err := cmd.Flags().GetString("token")
+	if err != nil {
+		return err
+	}
+
+	v := viper.GetViper()
```

Comment:
To get the viper instance I would suggest to load it using the `func LoadLocalConfig()` located in `pkg/config/config.go` file.

You can use the same argument values used in `cmd/root.go` at line 65 to understand how to populate it